cmd/okctl/deletetestcluster.go

```diff
@@ -165,13 +165,20 @@ $ kubectl get service --all-namespaces
 				return formatErr(err)
 			}
 
+			// TODO --- yeah, this is hardcoded, can we get it from store?
 			err = services.Cluster.DeleteCluster(o.Ctx, api.ClusterDeleteOpts{
 				ID: id,
+				FargateProfileName: "fp-default",
```

Comment:
Unless something has changed, `eksctl` takes care of deleting the fargate profile as is. If something has gone wrong, it is most likely because ALBs are still active

Reply:
I have read the source code. However, it never successfully actually manages to delete the cluster. If fargate profiles are removed up front. Then it works. :man_shrugging: